packages/terra-dropdown-button/tests/wdio/split-button-spec.js

```diff
@@ -175,6 +175,11 @@ Terra.describeViewports('Split Button', ['medium'], () => {
         browser.buttonUp();
         Terra.validates.screenshot('clicking outside closes dropdown');
       });
+
+      it('closes the dropdown when clicking primary button and selects primary button', () => {
+        browser.click('[class*=split-button-primary]');
```

Comment:
Don't we need to open the dropdown before clicking the primary button?

```suggestion
browser.click('[class*=split-button-caret]');
browser.waitForVisible('[class*=dropdown-list]');
browser.click('[class*=split-button-primary]');
Terra.validates.screenshot('clicking primary closes dropdown and selects primary');
```

Reply:
It looks like the two tests before this might not be valid. It doesn't appear that the dropdown is open before testing close funcitonality.

Reply:
> Don't we need to open the dropdown before clicking the primary button?
https://github.com/cerner/terra-core/blob/b7f0a5a14f82503b99cd2e4918bb1148ef1bba05/packages/terra-dropdown-button/tests/wdio/split-button-spec.js#L106-L110

The menu opening logic is written in beforeEach().